CONTRIBUTING.md

```diff
@@ -1,34 +1,44 @@
 # Contributing OpenELA Documentation
 
 We welcome your contributions! There are multiple ways to contribute.
-All contributions to this repository and to OpenELA should following
-Contribution Guide including our Code of Conduct, which are included
-by reference here:
-https://github.com/openela/governance/blob/main/contributors_guidelines.md
 
-## Did you find a bug?
+All contributions to this repository and to OpenELA in general follow
+[the OpenELA Code of Conduct](https://github.com/openela/governance/blob/main/code_of_conduct.md).
+
+
+## Did you find an issue?
+
+Issues, be it a defect, a missing documentation part or any enhancement request, please [open an issue on GitHub](https://github.com/openela/openela-documentation/issues). Please provide as much detail as you can
+provide, including a reference to the particular chapter or paragraph that the issue is referring to.
+
+The better written the issue is, the more likely it is going to get fixed.
 
-For bugs or enhancement requests, please open a GitHub issue. 
-The better written the bug is, the more likely it is to be fixed. 
 
 ## Contribute documentation
 
-This is a community project and we welcome community members to contribute 
-documentation to the project.
+This is a community project and we welcome community members to contribute
+documentation to the project. Contributions are done [by opening a pull request against the project](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+Any contribution requires a [Developer Certificate of Origin](https://developercertificate.org/) (a Signed-off-By tag in the commit message) and
+must adhere to the [project license](./LICENSE).
+
```

Comment:
Looks like lines 22 and 23 should be combined, as in there may  be a CR/LF after "and" on 22 that should be removed.

Reply:
actually multiline paragraphs is valid in markdown..